src/test/java/com/google/javascript/clutz/partial/missing_base.js

```diff
@@ -28,6 +28,18 @@ function D() {
 
 }
 
+/**
+ * @param {C!} c
```

Comment:
please use speaking names for all symbols,  future reviewers will praise their ability to understand what this is testing :-)